src/Sinks/Parquet/Exceptions/SchemaInconsistentException.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Arcane.Framework.Sinks.Parquet.Exceptions;
+
+/// <summary>
+/// Thrown if the schema of the source is inconsistent with the schema of the sink
```

Comment:
This can occur if the source updates the schema after the stream has been launched

Reply:
Added this to docsstring